README.md

```diff
@@ -73,10 +93,7 @@ cd ..
 
 ### 2. Set Up EventYay Video
 
-The video webapp needs node modules etc installed and build.
-This is done in during the docker image build step, but since
-we mount the checked out eventyay-video directory into the
-container, the built-in directory with node-modules etc is hidden.
+The video webapp needs node modules etc installed and build. This is done in during the docker image build step, but since we mount the checked out eventyay-video directory into the container, the built-in directory with node-modules etc is hidden. If doing this on mac make sure rosetta is used by terminal you can confirm this by using arch this should be i386
```

Comment:
Don't make one lon line, please leave it like it is.
Markdown does not care for newlines